frontend/src/components/TE/TaskDefinition.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <template v-for="(chunk, index) in item.chunks">
+      <v-layout class="wrap pb-1" :key="index">
+        <v-flex xs4 sm2 md2 xl1>
+          <div>
+            <a
+              download
+              :href="getLink(chunk.fileName)"
+            >
+              {{chunk.fileName}}
+            </a>
+          </div>
+        </v-flex>
+        <v-flex xs8 sm2 md2 xl1>
+          <span>{{ formatSeconds(chunk.beginning) }}&#8211;{{ formatSeconds(chunk.ending) }}</span>
+        </v-flex>
+        <v-flex sm12 md8 lg8 xl9>
+          <unwanted-parts :item="chunk" />
```

Comment:
Why not to pass unwanted parts directly?
```suggestion
          <unwanted-parts :item="chunk.unwantedParts" />
```